### DISCUSSION TITLE
Support for beta/nightly version?

### DISCUSSION THREAD

**Discussion by jsjoeio** [Homebrew/discussions#2217] [Writing Formulae/Casks]
Hi all! 👋

I'm one of the [code-server](https://github.com/cdr/code-server) maintainers and we're interested in providing beta/nightly releases to our homebrew users.

> Unstable versions (alpha, beta, development versions) are not acceptable for versioned (or unversioned) formulae.
> [Source](https://docs.brew.sh/Versions#acceptable-versioned-formulae)

To me, it sounds like this isn't supported in homebrew/core but maybe I'm reading this wrong.

I'm wondering how others do this?

Thanks in advance!

**Comment by SMillerDev** [comment]
You can maintain your own tap.

**Comment by gwerbin** [comment]
Maybe you can also support `HEAD` builds, if you have a VCS branch that tracks this nightly build.
